Event/include/Event/HgcrocTrigDigi.h

```diff
@@ -0,0 +1,152 @@
+#ifndef EVENT_HGCROCTRIGDIGI_H_
+#define EVENT_HGCROCTRIGDIGI_H_
+
+// ldmx-sw
+#include <stdint.h> //uint{32,8}_t
+
+// ROOT
+#include "TObject.h" //For ClassDef
+
+namespace ldmx {
+
+/**
+ * @class HgcrocTrigDigi 
+ * @brief Contains the trigger output for a single trigger hgcroc channel
+ */
+class HgcrocTrigDigi {
+ public:
+
+  /**
+   * Default Constructor
+   *
+   * Needed for ROOT dictionary definition,
+   * suggested to not use this constructor.
+   */
+  HgcrocTrigDigi();
+
+  /**
+   * Preferred Constructor
+   *
+   * Defines the trigger group ID and
+   * the trigger primitive value.
+   *
+   * @param[in] tid raw trigger group ID
+   * @param[in] tp trigger primitive value
+   */
+  HgcrocTrigDigi(uint32_t tid, uint8_t tp=0);
+
+  /**
+   * Destructor
+   *
+   * Needs to be defined for ROOT
+   * dictionary definition, does
+   * nothing right now.
+   */
+  virtual ~HgcrocTrigDigi();
```

Comment:
If this does nothing, then might as well set it equal to default 
`virtual ~HgcrocTrigDigi() = default;`

you can then remove the definition from the src file.